UI/Web/src/app/book-reader/_components/book-reader/book-reader.component.ts

```diff
@@ -202,6 +202,7 @@ export class BookReaderComponent implements OnInit, AfterViewInit, OnDestroy {
    * Used for showing/hiding bottom action bar. Calculates if there is enough scroll to show it.
    * Will hide if all content in book is absolute positioned
    */
+  columnGap: number = 20;
```

Comment:
Move this to a const outside the class since it's not changing. Then you don't need to access on this.